index.src.html

```diff
@@ -1049,6 +1062,20 @@ <h3 id="interfaces-credential-manager">Credential Manager</h3>
         credential: The credential to be stored.
       </pre>
     </dd>
+    <dt><dfn method>bad()</dfn></dt>
```

Comment:
Nit: `bad()` is a bad name. :) We have `get()` and `store()`, why not `remove()` or `delete()` or something along those lines?